app/models/group_milestone.rb

```diff
@@ -20,6 +20,17 @@ def milestones
   def projects
     milestones.map { |milestone| milestone.project }
   end
+  
+  def expires_at
+    due_date = milestones.map { |milestone| milestone.due_date }.max
+    if due_date
+      if due_date.past?
+        "expired at #{due_date.stamp("Aug 21, 2011")}"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.